### PR TITLE
Upgraded Make Some Noise with E-commerce Data

### PR DESCRIPTION
**Pull request by marcushyett-ph**
Added more events representing an e-commerce funnel. 
This will create more realistic data that's easier to work with and more representative of a real product.

Main Changes:
- More types of events (like an e-commerce site)
- Fire events in the form of a funnel (so we can render a realistic funnel)
- Mixing up referrers - so we can do breakdowns on source of traffic
- Added product_id so we can breakdown by specific hypothetical products in analysis
- Removed some of the superfluous fields and ones specific to tracking posthog

It works:
image

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,73 +1,122 @@
...
-            $viewport_width: 789,
-            distinct_id: userData.username,
-            ee_available: true,
-            email_service_available: false,
-            is_clickhouse_enabled: true,
-            is_demo_project: false,
-            posthog_version: '1.27.0',
-            realm: 'hosted-clickhouse',
-        })
+makeNoise = async () => {
```
Comment (mariusandra) [inline index.ts]:
```suggestion
const makeNoise = async () => {
```

File: index.ts [hunk 2]
```diff
@@ -1,73 +1,122 @@
...
+        referrer: "/",
+        current_url: eventTypes[i].url,
+        userData: userData,
+      });
     }
-}
+  }
+};
+
+submitEvent = async (event) => {
```
Comment (mariusandra) [inline index.ts]:
```suggestion
const submitEvent = async (event) => {
```

... or just 

```
async function submitEvent (event) {
```

**Review by mariusandra — APPROVED** [review]
Basically looks amazing, though I would strongly suggest running the code through `prettier` to make it, well, prettier. Added two comments as well that won't make a difference at the end of the day, but if we're nitpicking already...

And just in case a green checkmark so you can bias for getting it in.

**Comment by marcushyett-ph** [comment]
Thanks for your feedback @mariusandra

Hopefully it's looking prettier now too...

**Pull request merged**